- fix build on platforms where sanitizer isn't available: armv7, aarch64, ...

--- src/common/CMakeLists.txt.orig	2023-08-17 05:30:38 UTC
+++ src/common/CMakeLists.txt
@@ -33,8 +33,8 @@ if (BUILD_TESTS)
             $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/external/catch2>
             $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>)
     target_link_libraries(common_plugin_tests PRIVATE ${LIBRARIES})
-    target_compile_options(common_plugin_tests PRIVATE -fsanitize=undefined)
-    target_link_options(common_plugin_tests PRIVATE -fsanitize=undefined)
+    #target_compile_options(common_plugin_tests PRIVATE -fsanitize=undefined)
+    #target_link_options(common_plugin_tests PRIVATE -fsanitize=undefined)
 
     include(CTest)
     include(Catch)
